docs/reference/configuration/keywords.rst

```diff
@@ -790,16 +790,24 @@ and/or history matching project.
 .. _gen_data:
 .. topic:: GEN_DATA
 
-        The ``GEN_DATA`` keyword is used to load text files which have been generated
-        by the forward model.
+        The ``GEN_DATA`` keyword is used to declare text files that are referred to by
+        :ref:`GENERAL_OBSERVATIONs <general_observation>`. It declares
+        text files which have been generated by the forward model, which will be
+        loaded by ert when loading general observations.
+        These text files are expected to follow the same naming scheme for all realizations
+        (ex: ``gd_%d`` which may resolve to ``gd_0``, ``gd_1`` where `%d` is the report step).
+        The contents of these result are always of this format:
+        **exactly one floating point number per line**.
+        If we speak about "indexing" into a ``GEN_DATA`` file,
```

Comment:
Would reword this slightly, perhaps replacing: `If we speak about "indexing"` ->` Indexing ``GEN_DATA`` refers to the row number...`